EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/SqlServerMigrationTestBase.cs

```diff
@@ -176,6 +176,13 @@ protected List<string> FetchDatabaseReferencesJournalEntries()
                 .Where(se => se.DatabaseEngine == "MsSql" && se.IsFeature == false)
                 .Select(x => x.JournalScriptEntry)
                 .ToList();
+
+            foreach(var feature in FeaturesBeforeUpgrade)
+            {
+                databaseReferencesJournalEntries.AddRange(journalEntries.Where(se => se.DatabaseEngine == "MsSql" && se.IsFeature == true && se.Feature == feature.SubTypeFolderName)
```

Comment:
You can simplify `se.IsFeature == true` down to just `se.IsFeature` which would evaluate the same way. A similar change can be made in the PostgresSqlMigrationTestBase

Reply:
I fixed it . Please review again